docs/home/css/styles.css

```
@@ -1,7 +1,9 @@
-@import url('https://fonts.googleapis.com/css?family=Open+Sans|Roboto&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@100;300;400;500;700;900&display=swap');
 
 *{
   padding: 0;
+  color: #E5E5E5;
+  margin: 0;
 }
 
```

sugiro adicionar isso aqui para trabalhar com responsividade
```suggestion
 @media (max-width: 1080px){
    html {
        font-size: 93.75%;
    }
  }
  @media (max-width: 720px){
    html {
        font-size: 87.5%;
    }
  }
```